lib/lift_wing_api.rb

```diff
@@ -2,9 +2,9 @@
 
 require_dependency "#{Rails.root}/lib/errors/api_error_handling"
 
-# Gets data from Lift Wing
+# Gets and processes data from Lift Wing
 # https://wikitech.wikimedia.org/wiki/Machine_Learning/LiftWing
-class LiftWingApi
+class LiftWingApi # rubocop:disable Metrics/ClassLength
```

Comment:
I had to disable the rule because the class is too long.
One option could be to move the `WEIGHTING` constants outside the class? Other thing that would take more effort would be to split the `LfitWingApi` class in two different classes: one to get data form the api, and other one to process it (calculating the `wp10` for example).

Reply:
The better place to do this is `.rubocop.yml` so that we have a central list of all the too-big classes.